src/nvme/crc32.c

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: FSFAP-no-warranty-disclaimer
```

Comment:
My bad, but I think we can't add the SPDX tag because the license from Gary S. Brown doesn't show up in SPDX database. And the FSFAP is not the right one.

I tried to figure out how this handled by other project and found many copies of the crc32 code below. Some have added a GPL header in front (don't know how this should work but it's not our problem) and other project just use the crc32.c code with and without shortening the original code an don't mention any additional copyright statements.  

The Fedora project has a note on this https://fedoraproject.org/wiki/Licensing/CRC32

Given this, I'd say we don't add any SPDX header until the SPDX project has covered Brown's license.

I think we have done our homework on the license topic.

Reply:
I got once burned with not paying enough attention to license issues. So I am rather playing safe here. Sorry about that.

Reply:
Ok, removed the SPDX line here.